bpf/enhancedrecording/common.h

```diff
@@ -0,0 +1,3 @@
+
+// Maximum monitored sessions.
+#define MAX_MONITORED_SESSIONS 1024
```

Comment:
Should we add `#ifndef` for header here just in case?

Reply:
Added